### ISSUE TITLE
Consider porting to atom-shell or node-webkit

### ISSUE THREAD

**Issue by rjmunro** [simoncozens/songbee#31]
https://github.com/atom/atom-shell/blob/master/docs/development/atom-shell-vs-node-webkit.md

**Comment by simoncozens** [comment]
Yeah, I have a plan to port this to node-webkit. The main hurdle is the need for multiple-monitor support.
